tests/presenters/alert-presenter-test.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Yoast\WP\SEO\Tests;
+
+use Brain\Monkey;
+use Yoast\WP\SEO\Tests\TestCase;
+
+/**
+ * Unit Test Class.
+ *
+ * @coversDefaultClass \Alert_Presenter
+ */
+class Alert_Presenter_Test extends TestCase {
+
+	/**
+	 * Test when the Alert is of type 'error'.
+	 *
+	 * @covers ::render()
+	 * @covers ::__toString()
+	 */
+	function test_error_alert() {
+		Monkey\Functions\expect( 'wp_enqueue_style' )->once();
+
+		$test = new \Alert_Presenter(\Alert_Presenter::ERROR, 'content' );
+
+		$expected = '<div class="alert alert__error">'
+			. '<span><img class="icon" src="images/alert-error-icon.svg""></span>'
```

Comment:
The double quotes at the end of the img tag should be fixed once the related code in the tested method gets fixed.
An empty alt attribute should be added once the related code in the tested method gets fixed.